package.json

```diff
@@ -101,6 +102,7 @@
     "uuid": "^8.3.0",
     "winston": "^3.3.3",
     "winston-transport": "^4.4.0",
+    "ws": "^7.4.0",
```

Comment:
We're going to need `ws` in regular dependencies soon, so I didn't bother putting it in dev.